src/protocols/BaseProtocolAdapter.ts

```diff
@@ -9,4 +14,30 @@ export abstract class BaseProtocolAdapter {
   static derivationPath: string;
 
   abstract getBalance(address: string): Promise<string>;
+
+  /**
+   * Check if the network has seen this account
```

Comment:
I'm not still sture what actually "seen" means. Maybe you could describe it in more clear way?

Reply:
Changed it to `Check whether the network has encountered this account.` @peronczyk please check